eme_listeners.js

```diff
@@ -359,10 +359,10 @@ EmeListeners.extendEmeMethod = function(
     if (result && result.constructor.name == 'Promise') {
       var description = title + ' Promise Result';
       result = result.then(function(resultObject) {
-        EmeListeners.logPromiseResult(description, 'resolved', resultObject);
+        EmeListeners.logPromiseResult(description, 'resolved', resultObject, args);
```

Comment:
Do we really want to log all arguments to this function?  (element, originaFn, title, etc)

Why not log currentKeySystem instead, as logCall does on lines 357-358?

Reply:
Logging all arguments makes it easier to know which `MediaKeySystemConfiguration` objects were passed. See
![screenshot 2017-09-07 at 9 29 06 am](https://user-images.githubusercontent.com/634478/30151288-1bad4a16-93af-11e7-8296-bbf6d65a76db.png)

Reply:
You're right. I'll fix now.